src/constants/index.ts

```diff
@@ -655,3 +655,39 @@ export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
 // FLAGS
 export const IS_VIRTUAL_SWAP_ACTIVE = false
 // FLAGS END
+
+export const EVM_CHAINIDS = {
+  ETHEREUM: 1,
+  BSC: 56,
+  MATIC: 137,
+  AVALANCHE: 43114,
+  FANTOM: 250,
+  ARBITRUM: 42161,
+  MOONRIVER: 1285,
+  BOBA: 288,
+  HARMONY: 1666600000
+}
+
+export const RPC_URLS = {
+  [EVM_CHAINIDS.ETHEREUM]: 'https://mainnet.infura.io/v3/',
+  [EVM_CHAINIDS.BSC]: 'https://bsc-dataseed.binance.org/',
+  [EVM_CHAINIDS.MATIC]: 'https://rpc-mainnet.maticvigil.com/',
+  [EVM_CHAINIDS.AVALANCHE]: 'https://api.avax.network/ext/bc/C/rpc',
+  [EVM_CHAINIDS.FANTOM]: 'https://rpc.ftm.tools/',
+  [EVM_CHAINIDS.ARBITRUM]: 'https://arb1.arbitrum.io/rpc',
+  [EVM_CHAINIDS.MOONRIVER]: 'https://moonriver.api.onfinality.io/public',
+  [EVM_CHAINIDS.BOBA]: 'https://mainnet.boba.network',
+  [EVM_CHAINIDS.HARMONY]: 'https://api.harmony.one',
+}
+
+export const FRAX_ADDRESSES: { [key: string]: string }  = {
+  [EVM_CHAINIDS.ETHEREUM]: '0x853d955aCEf822Db058eb8505911ED77F175b99e',
+  [EVM_CHAINIDS.BSC]: '0x90C97F71E18723b0Cf0dfa30ee176Ab653E89F40',
+  [EVM_CHAINIDS.MATIC]: '0x45c32fA6DF82ead1e2EF74d17b76547EDdFaFF89',
+  [EVM_CHAINIDS.AVALANCHE]: '0xD24C2Ad096400B6FBcd2ad8B24E7acBc21A1da64',
```

Comment:
The token address is wrong, AnyFRAX is 0xDC42728B0eA910349ed3c6e1c9Dc06b5FB591f98

Reply:
@prodvdjin  if this is the wrong token, How are you even testing to make sure it works?